examples/managed_sans.py

```diff
@@ -46,25 +46,25 @@
 
 # Get all, with defaults
 print("\nGet all Managed SANs")
-managed_sans = oneview_client.managed_sans.get_all()
+managed_sans_all = managed_sans.get_all()
 
-if managed_sans:
-    for managed_san in managed_sans:
+if managed_sans_all:
+    for managed_san in managed_sans_all:
         print('  Name: {name} - Manager: {deviceManagerName} - Imported: {imported}'.format(**managed_san))
         if managed_san['imported']:
-            imported_san = managed_san
+            imported_san = managed_sans.get_by_uri(managed_san["uri"])
         else:
-            internally_managed_san = managed_san
+            internally_managed_san = managed_sans.get_by_uri(managed_san["uri"])
 
     # Get a Managed SAN by name
     print("\nGet a Managed SAN by name")
-    managed_san_by_name = oneview_client.managed_sans.get_by_name(managed_san['name'])
-    pprint(managed_san_by_name)
+    managed_san_by_name = managed_sans.get_by_name(managed_san['name'])
+    pprint(managed_san_by_name.data)
```

Comment:
how the data was printing earlier ? why it is changed from 'managed_san_by_name ' to ' managed_san_by_name .data'   now ?

Reply:
This is because of the Python restructuring. 
get_by_name or get_by_uri will return an object of the resource and the data of the resource will be with the data attribute.